php/fomo/config/server.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+return [
+    'mode' => SWOOLE_BASE ,
+    'host' => '127.0.0.1',
+    'port' => 9000 ,
+    'sockType' => SWOOLE_SOCK_TCP ,
+    'additional' => [
+        'worker_num' => cpuCount() * 2 ,
```

Comment:
maybe

```suggestion
        'worker_num'       => swoole_cpu_num() * 2,
```

will be more accurate